README.md

```diff
@@ -1,57 +1,52 @@
-# Resource dispatcher OCI image
+# Resource dispatcher ROCK OCI image
 
-This repository contains an image with an HTTP server that should be used in the Kubernetes meta-controller charm called [resource-dispatcher](https://github.com/canonical/resource-dispatcher). For each GET request on the /sync path, the resource dispatcher will generate Kubernetes manifests that need to be injected into the given Kubernetes namespace. Information about the Kubernetes namespace should be included in the body of the request. Here is an example of a JSON body:
+The following tools are required to build ROCK image manually:
+- `rockcraft' - A tool to create OCI images.
+- `skopeo` - A tool to operate on container images and registries.
 
+To install tools:
 ```
-{
-    "parent": {
-        "metadata": {
-            "name": "someName",
-            "labels": {
-                "user.kubeflow.org/enabled": "true"
-            }
-        }
-    },
-    "children": {
-        "Secret.v1": [],
-    }
-}
+sudo snap install rockcraft --classic --edge
+sudo snap install skopeo --edge --devmode
```

Comment:
Should we explicitly ask for `skopeo` installation?

When you install `rockcraft`, `skopeo` is included in the package.


```shell
ubuntu@docker:~$ snap list
Name             Version        Rev    Tracking       Publisher   Notes
bare             1.0            5      latest/stable  canonical✓  base
core18           20230426       2745   latest/stable  canonical✓  base
core20           20230404       1879   latest/stable  canonical✓  base
lxd              5.0.2-838e1b2  24322  5.0/stable/…   canonical✓  -
multipass-sshfs  1.1            145    latest/stable  canonical✓  -
rockcraft        0+git.e4acb2a  737    latest/edge    sergiusens  classic
snapd            2.57.5         17576  latest/stable  canonical✓  snapd
yq               v4.33.3        2178   latest/stable  mikefarah   -


ubuntu@docker:~$ whereis skopeo
skopeo: /usr/bin/skopeo /usr/share/man/man1/skopeo.1.gz
```